dubbo-metadata-report/dubbo-metadata-report-etcd/src/test/java/org/apache/dubbo/metadata/store/etcd/EtcdMetadataReportTest.java

```diff
@@ -73,6 +74,7 @@ public void tearDown() throws Exception {
     }
 
     @Test
+    @Ignore("Ignore because https://github.com/apache/dubbo/issues/4185")
```

Comment:
In junit5 ， `Ignore ` is  changed to `Disabled`

Reply:
Fixed